ercc/main.go

```diff
@@ -34,39 +34,42 @@ func main() {
 		log.Panicf("error create enclave registry chaincode: %s", err)
 	}
 
-	// start chaincode as a service
-	config := serverConfig{
-		CCID:    os.Getenv("CHAINCODE_PKG_ID"),
-		Address: os.Getenv("CHAINCODE_SERVER_ADDRESS"),
-	}
+	ccid := os.Getenv("CHAINCODE_PKG_ID")
+	addr := os.Getenv("CHAINCODE_SERVER_ADDRESS")
 
-	server := &shim.ChaincodeServer{
-		CCID:    config.CCID,
-		Address: config.Address,
-		CC:      ercc,
-		TLSProps: shim.TLSProperties{
-			Disabled: true,
-		},
-	}
```

Comment:
This is good; instead of wrapping the code that comes after in an `if` we could just panic/exit here if `ccid` and `addr` are not set. This makes the change set smaller and code easier to read.

Reply:
I would agree but cannot panic (check out the code below): if those parameters are not set, the code still proceeds with the original initialization.

Reply:
> instead of two if might be more robust to do a if .. elsif .. else ..

fixed